src/Components/RocketItem.js

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { updateReserve } from '../Redux/rockets/RocketReducer';
+
+const RocketItem = (props) => {
+  const { rocket } = props;
+  const dispatch = useDispatch();
+  const clickHandler = () => {
+    dispatch(updateReserve(rocket.id));
+  };
+
+  return (
+    <li className="grid grid-cols-1 gap-3 sm:grid-cols-2">
+      <div>
+        <img src={rocket.image} alt={`rocket-${rocket.id}`} />
+      </div>
+      <div className="flex flex-col gap-3 justify-items-start">
+        <h2 className="text-2xl font-bold font-important">{rocket.name}</h2>
+        <p className="text-base font-normal">{rocket.description}</p>
+        <div className="text-base font-normal">
+          { (!rocket.reserved) && <button onClick={clickHandler} type="button" className="bg-blue-600 text-white p-2 rounded hover:bg-white hover:text-blue-600">Reserve Rocket</button>}
+          {(rocket.reserved) && <button onClick={clickHandler} type="button" className="bg-gray-500 text-black p-2 rounded hover:bg-blue-600 hover:text-white">Cancel Reserve Rocket</button>}
+        </div>
+      </div>
+    </li>
```

Comment:
- [ ] @team great work on your rockets however the design differs slightly from the design given, the cancel reserve button color is different also the reserved logo is missing, check the comparison below:

Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/209117532-048bca55-e7b3-4f3f-a4af-7d83e41d8b5b.png) |  ![image](https://user-images.githubusercontent.com/61408860/209117394-243f5f83-cb7e-46ca-aad1-652bb8097299.png)


- [ ] Also I noted your line is missing the line separating the navbar from the rockets and the profile page
![image](https://user-images.githubusercontent.com/61408860/209117328-1734a718-8ef5-4e92-b3a8-6cb47801cb32.png)